server/eit/commands/inject.py

```diff
@@ -51,6 +52,9 @@ def _get_parser(self):
         parser.add_argument("--quick", action="store_true",
                             default=not self._ask,
                             help=_("no stupid questions"))
+        parser.add_argument("--reset-revision", action="store_true",
+                            default=self._reset_revision,
+                            help=_("Reset revision to 0."))
```

Comment:
- use the same punctuation (capital, dot)
- consider adding "(advanced)" at the end

Reply:
Do you want a comment like this: `reset revision to 0` (without dot and capital) ?
Do you want add `advanced`where ? in the option or in the help message ?

Reply:
Yes, `reset revision to 0`. About the second thing, I meant `reset revision to 0 (advanced)` but in the end, maybe that `(advanced)` is not that useful to add.